### PR TITLE
sdn: add metric for VNID map error latency

### PR DESCRIPTION
**Pull request by dcbw**
@eparis @danwinship @openshift/networking

### CODE REVIEW

**Comment by knobunc** [comment]
@openshift/networking PTAL

**Comment by eparis** [comment]
anything that needs cleanup worries me. given we have no more time in case there is a bug here, I vote we don't merge until next release.

File: pkg/network/node/vnids.go [hunk 1]
```diff
@@ -175,9 +189,27 @@ func (vmap *nodeVNIDMap) Start() error {
 	}
 
 	go utilwait.Forever(vmap.watchNetNamespaces, 0)
+	go utilwait.Forever(vmap.cleanNotFoundMap, 0)
 	return nil
 }
 
+// cleanNotFoundMap() removes any VNID-not-found timestamps that are older than 10 minutes
+// to ensure we don't leave stale entries in the map that will never be found
```
Comment (danwinship) [inline pkg/network/node/vnids.go]: This means that if the notfound latency goes beyond 10 minutes, we end up reporting the latency mod 10. (And wasn't it getting up to at least 8 minutes before?)

Stale entries should be pretty rare. We can afford to do the cleanup much less frequently

**Comment by pravisankar** [comment]
@dcbw @danwinship 
My hunch is current low timeout is causing the issue. https://github.com/openshift/origin/pull/17243 may fix this issue. VNIDNotFoundLatencyKey and cleanNotFoundMap go routine may be overkill at this point. Can we just collect VNIDNotFoundErrorsKey metric for now and if this issue persists after #17243 , we could introduce this latency/NotFoundMap code?

**Comment by danwinship** [comment]
> My hunch is current low timeout is causing the issue. #17243 may fix this issue.

@pravisankar This PR isn't actually about a VNID problem, it's just that when the iptables lossage happens, it causes cascading failures, including horribly-delayed NetNamespace notifications (due to goroutines unexpectedly getting blocked waiting for the iptables lock), so this is a useful way to measure some of the lossage

**Comment by dcbw** [comment]
just going to close this one for now; can re-open if we need it in the future.

**Pull request closed without merging**